ui/src/common/components/CollaborationLink.tsx

```diff
@@ -0,0 +1,13 @@
+import React, { Component } from 'react';
+import { Link } from 'react-router-dom';
+
+import { LITERATURE } from '../routes';
+
+const CollaborationLink = ({children}: {children: string}) => {
+  const collaboration = children;
+  const link = `${LITERATURE}?q=collaboration:${collaboration}`;
+  return <Link to={link}>{collaboration}</Link>;
```

Comment:
We could either 

```suggestion
  const link = `${LITERATURE}?q=collaboration:${children}`;
  return <Link to={link}>{children}</Link>;
```

or 

```ts
const CollaborationLink = ({children: collaboration}: {children: string})
```

wdyt?